src/main/java/grumblebug/Storage.java

```diff
@@ -0,0 +1,82 @@
+package grumblebug;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.Scanner;
+
+/**
+ * Class represents the object used to deal with storage into and out of the local file.
+ */
+public class Storage {
+    // load data from hard disk
+
+    /**
+     * Load any previously stored task data into the current taskList.
+     * @param filePath The path to the storage file.
+     * @param taskList The taskList to store this data into.
+     */
+    public static void loadFromFile(String filePath, TaskList taskList) {
+        Parser parserStorage = new Parser("yyyy-MM-dd");
+        try {
+            File f = new File(filePath);
+            f.createNewFile();
+            Scanner s = new Scanner(f);
```

Comment:
Should the variable name here for _s_ be longer and more descriptive especially since the scope it is used in is quite large?

> Variables with a large scope should have long names, variables with a small scope can have short names.